migrations/15_Configure_Oracle_and_set_Avatar_as_DX_auctioneer.js

```diff
@@ -0,0 +1,57 @@
+
+/* global artifacts, web3 */
+/* eslint no-undef: "error" */
+
+module.exports = async function (deployer, network) { // eslint-disable-line no-unused-vars
+  console.log('network: ', network);
+
+  if (process.env.USE_FIXED_PRICE_ORACLE) {
+    const FixedPriceOracle = artifacts.require('FixedPriceOracle')
+
+    const PriceOracle = await FixedPriceOracle.deployed()
+
+    let whitelistedTokens
+    if (network === 'rinkeby') {
+      // Add some test tokens for Rinkeby    
+      whitelistedTokens = {
+        WETH: '0xc778417e063141139fce010982780140aa0cd5ab',
+        RDN: '0x3615757011112560521536258c1e7325ae3b48ae',
+        OMG: '0x00df91984582e6e96288307e9c2f20b38c8fece9',
+        testGEN: '0xa1f34744c80e7a9019a5cd2bf697f13df00f9773',
+        GEN: '0x543ff227f64aa17ea132bf9886cab5db55dcaddf'
+      }
+    } else if (network.startsWith('mainnet')) {
+      whitelistedTokens = {
+        GNO: '0x6810e776880C02933D47DB1b9fc05908e5386b96',
+        GEN: '0x543ff227f64aa17ea132bf9886cab5db55dcaddf'
+      }
+    }
+    console.log('\nSetting price for tokens on FixedPriceOracle: ')
+    let tokens = [], nums = [], dens = []
+    Object.keys(whitelistedTokens).forEach((tokenName, i) => {
+      const address = whitelistedTokens[tokenName]
+      console.log('  - %s: %s', tokenName, address, `to 1/${i+1}`)
+
+      tokens.push(address)
+      nums.push(1)
+      dens.push(i + 1)
+    })
+
+    if (tokens.length > 0) {
+      await PriceOracle.setPrices(tokens, nums, dens)
```

Comment:
Setting price as 1/1 for GNO and 1/2 for GEN

Reply:
Who is the owner of the FixPriceOracle ? I thought the plan is to use daostack priceOracleMock  , which will be owned by a multisg

Reply:
Deploying account is the owner.

The plan was to use fix price oracle, unless it was changed afterwards. The bounty doc specifically mentions a fix oracle, same as was used before.